Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java

```diff
@@ -137,6 +118,27 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     }
   }
 
+  private void setInsets()
+  {
+    ViewCompat.setOnApplyWindowInsetsListener(mBinding.getRoot(), (v, windowInsets) ->
+    {
+      Insets cutInsets = windowInsets.getInsets(WindowInsetsCompat.Type.displayCutout());
+      int left = 0;
+      int right = 0;
+      if (ViewCompat.getLayoutDirection(mBinding.getRoot()) == ViewCompat.LAYOUT_DIRECTION_LTR)
+      {
+        left = cutInsets.left;
+      }
+      else
+      {
+        right = cutInsets.right;
+      }
+      mBinding.getRoot().setPadding(left, cutInsets.top, right,
+              cutInsets.bottom + InsetsHelper.dpToPx(requireContext(), 16));
```

Comment:
Adding this value as an integer resource would save us from needing this `dpToPx` function and feels less like hardcoding values.